src/Endpoints/AssetFile.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Chromatic\OrangeDam\Endpoints;
+
+class AssetFile extends Endpoint
+{
+    /**
+     * API base path.
+     *
+     * @var string
+     */
+    protected const API_BASE_PATH = '/webapi/mediafile/assetfile/';
+
+    /**
+     * Create links to formatted assets.
+     *
+     * @param string $format
+     *   An Orange Dam asset format, such as TR1, or TR1_WATERMARKED.
+     * @param string $sort
+     *   The sort order of found assets. If you are using an identifer, the
+     *   result should just be one listing. The API default is 'Newest First'.
+     * @param array $queryConditions
+     *   An array of searchkeys and searchvalues with which to perform a Search
+     *   API syntax freetext search.
+     *   Example: ['docType' => 'Image',  'Test.UserFieldA' => 'ZZZ']
+     *
+     * @return \Chromatic\OrangeDam\Http\Response
+     */
+    public function getFormatFile(string $format, string $sort = '', array $queryConditions = [])
+    {
+        // @todo Decide whether we should be validating the sort keys and
+        //   the formats against retrieved or static lists.
+        $query_string = "Format=$format";
+        if ($sort) {
+            $query_string .= '&Sort=' . rawurlencode($sort);
```

Comment:
Could we use this somewhere instead of manual string manipulation?

- https://www.php.net/manual/en/function.http-build-query.php

Reply:
> Could we use this somewhere instead of manual string manipulation?
> 
>     * https://www.php.net/manual/en/function.http-build-query.php

Today I learned that you actually _can_ generate RFC3986 query encoding with that function! I'll try it.